jupytercad/freecad/loader.py

```diff
@@ -50,14 +96,20 @@ def load(self, base64_content: str) -> None:
         if not fc:
             return
         self._sources = base64_content
-        with tempfile.NamedTemporaryFile(delete=False, suffix='.FCStd') as tmp:
+        with tempfile.NamedTemporaryFile(delete=False, suffix=".FCStd") as tmp:
             file_content = base64.b64decode(base64_content)
             tmp.write(file_content)
 
+        # Get objects
         fc_file = fc.app.openDocument(tmp.name)
         for obj in fc_file.Objects:
             self._objects.append(self._fc_to_jcad_obj(obj))
+
+        # Get metadata
         self._metadata = fc_file.Meta
+
+        self._options = _guidata_to_options(OfflineRenderingUtils.getGuiData(tmp.name))
```

Comment:
We might need `_options` for other kinds of data. Should we store the GUI data in the `gui` key of the `option` dict?

Reply:
Sounds good 👍🏽